File: 0217-contains-duplicate/0217-contains-duplicate.js
/**
 * @param {number[]} nums
 * @return {boolean}
 */
var containsDuplicate = function(nums) {
    nums.sort((a,b)=>a-b)
    console.log(nums)
    for(let i in nums){
        if(nums[i] == nums[i-1] || nums[i] == nums[i+1])
            return true
    }
    return false
};

File: 0217-contains-duplicate/0217-contains-duplicate.cpp
class Solution {
public:
    bool containsDuplicate(vector<int>& nums) {
        unordered_set<int>distinctNums;
        for(int num : nums)
        {
            if(distinctNums.count(num)){
                return true;
            }
            distinctNums.insert(num);
        }
        return false;
    }
};